docs/reference/contributing/guidelines/workflow.md

```diff
@@ -109,11 +109,11 @@ Updating target implementation (adding a new target or updating already supporte
 
 Release: patch
 
-#### Feature
+#### Functionality change
 
-New features target feature releases. A new feature can only be integrated if the feature supports most of the targets (if it requires new target HAL implementation).
+Any change in the functionality, it can be adding a new feature, adding a new method or a function. It does not matter if it is C++, C or Python.
```

Comment:
```suggestion
Any change in the functionality, it can be adding a new feature, adding a new method or a function. Software language does not matter.
```

Reply:
What about changes required to bring in new targets?